azerothcore/azerothcore-wotlk
# feat(Core/Mmaps): Handle different slopes in  mmaps and Add some more input parameters to improve

## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## SOURCE:

https://github.com/TrinityCore/TrinityCore/commit/b6e205f7ef32f7b82644043d82a808e67cc9acb2
https://github.com/TrinityCore/TrinityCore/commit/3947e4cb57d4c6adb78cd3cb52718bb88add91fc
https://github.com/TrinityCore/TrinityCore/commit/995a443da219ec773febd7dd29d18f3cefaa1f3b
https://github.com/TrinityCore/TrinityCore/commit/7b561373a892c74870a21cf31687df4afd554495

## Tests Performed:

- Need Test again
- Re extract needed
- Part of multithreading mmaps


## How to Test the Changes:


1.Please read the source description
2. test with this pr https://github.com/azerothcore/azerothcore-wotlk/pull/10922
3.make sure blink, warrior charge works fine Especially in the arena

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
